.eslintrc

```diff
@@ -1,4 +1,3 @@
-extends: angular
```

Comment:
I think we should keep this. We shouldn't be using JQuery or console.log in e2e tests, we should be using angular.element and $log instead. I fixed those lint errors in my latest PR.